core/vm/actor/builtin/v0/init/init_actor_state.hpp

```diff
@@ -7,26 +7,6 @@
 
 #include "vm/actor/builtin/states/init_actor_state.hpp"
 
-#include "codec/cbor/streams_annotation.hpp"
-
 namespace fc::vm::actor::builtin::v0::init {
-  using primitives::address::Address;
-
-  struct InitActorState : states::InitActorState {
-    outcome::result<Address> addActor(const Address &address) override;
-    outcome::result<boost::optional<uint64_t>> tryGet(
-        const Address &address) override;
-  };
-  CBOR_TUPLE(InitActorState, address_map_0, next_id, network_name)
+  using states::InitActorState;
```

Comment:
For your approach these files are not needed anymore, but I'm not sure that we want to use different approaches for different states.
Probably, the common approach is more useful for sustain.
In this case we can just keep `using InitActorState = states::InitActorState`